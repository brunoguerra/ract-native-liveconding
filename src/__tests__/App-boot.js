import 'react-native';
import React from 'react';
import Intro from '../';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Intro />
  ).toJSON();
  expect(tree).toMatchSnapshot();
  expect(typeof tree).toBe('object');
});
