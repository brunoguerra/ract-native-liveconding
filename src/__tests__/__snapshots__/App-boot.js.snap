exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgb(142, 68, 173)",
      "flex": 1,
      "justifyContent": "center"
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontSize": 36,
        "fontWeight": "bold"
      }
    }>
    Counter with Redux
  </Text>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row"
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function bound touchableHandleResponderGrant]}
      onResponderMove={[Function bound touchableHandleResponderMove]}
      onResponderRelease={[Function bound touchableHandleResponderRelease]}
      onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
      onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
      onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent"
          },
          undefined
        ]
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "rgb(100, 28, 133)",
            "color": "white",
            "fontSize": 24,
            "fontWeight": "bold",
            "padding": 10
          }
        }>
        Counter 
        1
      </Text>
    </View>
  </View>
</View>
`;
